apps/whale-api/src/module.model/vault.auction.batch.history.ts

```diff
@@ -63,7 +63,7 @@ export interface VaultAuctionBatchHistory extends Model {
 
   vaultId: string
   index: number
-  from: string
+  from: string // -------------------| hex address
```

Comment:
```suggestion
  from: string // -------------------| hex script
```

Reply:
Address in "hex" form is called "Script".

Reply:
updated